samples/Pan/ca.pan

```diff
@@ -0,0 +1,235 @@
+#
+# Generated by RepositoryTask on 11/10/14 12:21 PM
```

Comment:
Sure! I've removed all but one of this type as they are normally produced by automated tasks.